tests/tools/oecertdump/README.md

```diff
@@ -0,0 +1,13 @@
+oecertdump
+=====
+
+oecertdump is a utility that generates and validates report and certificates.
```

Comment:
grammar: change "report" to either "a report" or "reports"

Reply:
Fixed